build-logic/src/main/kotlin/dokkabuild/testing/android-setup.gradle.kts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+package dokkabuild.testing
+
+/**
+ * Utilities for preparing Android projects for Integration Tests
+ */
+
+plugins {
+    base
+}
+
+val localAndroidSdkRoot: DirectoryProperty = objects.directoryProperty().apply {
+    convention(layout.projectDirectory.dir("projects/ANDROID_SDK"))
+}
+
+/**
+ * Create an `local.properties` file so that Android projects will automatically work in integration tests.
+ */
+abstract class CreateAndroidLocalPropertiesFile : DefaultTask() {
```

Comment:
Why do we need this task, as in the end we will just copy the text from this file? Can't we just have a property `androidSdkDirPath` which we will then use?

Reply:
Good point, I've updated the task to do both in one. (I originally adapted it from a Dokkatoo task which needed the separate tasks.)